app/import.go

```diff
@@ -817,6 +817,12 @@ func (a *App) ImportUserTeams(user *model.User, data *[]UserTeamImportData) *mod
 			}
 		}
 
+		if defaultChannel, err := a.GetChannelByName(model.DEFAULT_CHANNEL, team.Id); err != nil {
+			return err
+		} else if _, err = a.AddUserToChannel(user, defaultChannel); err != nil {
```

Comment:
I've changed this to use the `AddUserToChannel` public method instead of `addUserToChannel` private method.

Reply:
I think this should be using `addUserToChannel` as that bypasses the notification code.

Reply:
Well, but we can use the import feature while the system is running and the users are working, I think make sense to emit the websocket event (I'm not sure we are emitting it anyway because is another process).

Reply:
You are right, the websocket notification doesn't matter, but I'd still rather be consistent with the rest of the import code and use the private function - that's specifically why the private function exists - to allow the importer to bypass the bits in the public function that are redundant.

Reply:
Sounds good, I going to change it :)

Reply:
Done! :)